src/main/java/org/reaktivity/command/log/internal/LogCommand.java

```diff
@@ -54,27 +55,41 @@ public static void main(String[] args) throws Exception
             String directory = cmdline.getOptionValue("directory");
             boolean verbose = cmdline.hasOption("verbose");
             String type = cmdline.getOptionValue("type", "streams");
+            int interval = Integer.parseInt(cmdline.getOptionValue("interval", "0"));
```

Comment:
Adding `final int interval = Integer.parseInt(cmdline.getOptionValue("interval", "0"))` will make it easier on the compiler optimization in the suggested loop below